### PR TITLE
added missed mapdb itests

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: itests/pom.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
   <name>openHAB Add-ons :: Integration Tests</name>
 
   <modules>
+    <module>org.openhab.persistence.mapdb.tests</module>
```
Comment (cpmeister) [inline itests/pom.xml]: Why is this the only integration test here?

Comment (kaikreuzer) [reply itests/pom.xml]: Because we don't have any other - they didn't exist in openHAB 1 and that's where all (but this) persistence services came from.

Comment (kaikreuzer) [reply itests/pom.xml]: Note that you are looking at the master branch here, not at 2.5.x!

Comment (cpmeister) [reply itests/pom.xml]: I knew I was looking at master, but I didn't know that master got rid of all the integration tests!

Comment (kaikreuzer) [reply itests/pom.xml]: It got rid off ALL the add-ons (until the moment that we do a bulk move from the 2.5.x branch to master later this year).

**Pull request merged**